CGE-mlst/Dockerfile

```diff
@@ -0,0 +1,61 @@
+FROM debian:stretch
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="mlst"
+LABEL software.version="2.19.0"
+LABEL description="CGE Multi-Locus Sequence Typing 2.0"
+LABEL website="https://cge.cbs.dtu.dk/services/MLST/"
```

Comment:
could you also add a label for the bitbucket repo? `LABEL code.repository="https://bitbucket.org/genomicepidemiology/mlst/src/master/"`